scripts/eachdist.py

```diff
@@ -466,28 +465,6 @@ def install_args(args):
             check=True,
         )
 
-    allfmt = "-e 'file://{}" if args.editable else "'file://{}"
-    # packages should provide an extra_requires that is named
-    # 'test', to denote test dependencies.
-    extras = []
-    if args.with_test_deps:
```

Comment:
I think you should be dropping the argument itself, see line 198

Reply:
Fixed it in the new commit, thank you.